sources/platform/actors/running/actor_standby.md

```diff
@@ -0,0 +1,62 @@
+what parameters they accept, and what they return in the Actor README.
+
+To use the Actor in the Standby mode, you do not need to do anything, no clicking start button, etc. Simply use the provided hostname and endpoint in your application,
+hit the API endpoint, and consume results.
+
+![Standby tab](./images/actor_standby/standby-tab.png)
+
+#### Can I still run the Actor in normal mode?
+Yes, you can still modify the input, and click the Start button to run the Actor in the normal mode. However, node that the Standby Actor might
+not support this mode, the run might fail, or return empty results. The normal mode is always supported in the Standby Beta, even for Actors that do not handle
+it well. Please, head to Actor README to learn more about the capabilities of your chosen Actor.
+
+#### Is there any scaling to accommodate the incoming requests?
+When you use the Actor in the Standby mode, the system automatically scales the Actor to accommodate the incoming requests. Under the hood,
+the system starts new Actor runs, which you will see in the Actor runs tab, with origin set to Standby.
+
+#### How do I customize standby configuration?
+The Standby configuration currently consists of the following properties
+- **Max requests per run** - The maximum number of concurrent HTTP requests a single standby Actor run can accept. If this limit is exceeded, the system starts a new Actor run to handle the request, which will take a few seconds.
+- **Desired requests per run** - A desired number of concurrent HTTP requests a single standby Actor run can handle. If this limit is exceeded, the system preemptively starts a new Actor run to handle additional requests.
+- **Memory (MB)** - The amount of memory (RAM) allocated for the Actor standby run, in megabytes. With more memory, the Actor can typically handle more requests in parallel, but the number of compute units consumed and the cost also grow.
+- **Idle timeout (seconds)** - If a standby Actor run doesn’t receive any HTTP requests in this amount of time, the system will finish the run. When a new request comes, the system might need to start a new standby Actor run to handle it, which will take a few seconds. Higher idle timeout increases the responsiveness but increases the costs as the Actor needs to keep running longer.
+
+You can see these in the Standby tab of the Actor detail page. However, note that these properties are not configurable at the Actor level. If you wish to
+use the Actor-level hostname, this will always use the default configuration. In order to override this configuration, just create a new Task from the Actor.
+You can then head to the Standby tab of the created Task and modify the configuration as needed. Note that the task has specific hostname, so make
+sure to use that in your application if you wish to use the custom configuration.
+
+#### Are the Standby runs billed differently?
+No, the standby runs are billed in the same fashion as the normal runs.
```

Comment:
I would put a note here that the Standby Actors might incur unexpected costs because they're running in the background, waiting for requests.
```suggestion
No, the standby runs are billed in the same fashion as the normal runs.
However, there might be unexpected costs for running Actors in the Standby mode, as the Actors are running in the background and consuming resources even when no requests are being sent, until they are terminated after the idle timeout period.
```

Reply:
* => Standby runs ?